.github/workflows/ci.yml

```diff
@@ -11,111 +11,150 @@ permissions:
   contents: read
 
 jobs:
-  test:
```

Comment:
i am assuming that there won't really be any changes in this file, and you are just copying release-native for testing purposes.